BUILDING.md

```diff
@@ -87,6 +87,12 @@ The recommended generator on Windows is `Visual Studio 15 2017 Win64`:
 $ cmake .. -G "Visual Studio 15 2017 Win64" -Thost=x64
 ```
 
+Visual Studio 2019 is also supported. For this generator you must leave off the Win64:
+
+```console
+$ cmake .. -G "Visual Studio 16 2019" -Thost=x64
```

Comment:
Yes, Visual Studio 2019 uses a newer version of Platform Toolset (v142 instead of 141). I'm not suggesting we change the CI. This is just in case devs want to use the newer IDE and tools. All the tests pass, however.